sources/operator.1.0.0.pg-operator.json

```diff
@@ -0,0 +1,136 @@
+{
+  "versions": [
+    {
+      "operator": "1.0.0",
+      "product": "pg-operator",
+      "matrix": {
+        "postgresql": {
+          "12.8": {
+            "image_path": "percona/percona-postgresql-operator:1.0.0-ppg12-postgres-ha",
+            "image_hash": "ee234eef5f5e16f2eeaa894a1ab4593f52dcdc7507bd64245747d6617756ca26",
+            "status": "available",
+            "critical": false
+          },
+          "13.4": {
+            "image_path": "percona/percona-postgresql-operator:1.0.0-ppg13-postgres-ha",
+            "image_hash": "61ee6450cf35ff6267ba766ec3ffe4b67cff483b449be9d5571fe5b66a4a739c",
+            "status": "available",
+            "critical": false
+          }
+        },
+        "pmm": {
+          "2.15.1": {
+            "image_path": "percona/pmm-client:2.15.1",
+            "image_hash": "43108a08158006340a97378a79bedb56019743f9bbe01fbcdc7d058a36076c68",
+            "status": "recommended",
+            "critical": false
+          }
+        },
+        "pgbackrest": {
+          "12.8": {
+            "image_path": "percona/percona-postgresql-operator:1.0.0-ppg12-pgbackrest",
+            "image_hash": "1d8699e1820bc656a6fcae1bb3baac3912a6c917b27a5b57a3b539a85c0f5273",
+            "status": "available",
```

Comment:
we need to support 12-recommended, 13-recommended and 14-recommended options
please check it